tox.ini

```diff
@@ -793,6 +793,15 @@ envdir   = {[sage_src]envdir}
 commands = {[sage_src]commands}
 allowlist_externals = {[sage_src]allowlist_externals}
 
+[testenv:coverage.py]
+description =
+    run the Sage doctester with Coverage.py
+    (https://coverage.readthedocs.io/en/latest/index.html)
+passenv  = {[sage_src]passenv}
+envdir   = {[sage_src]envdir}
+commands = {[sage_src]commands}
+allowlist_externals = {[sage_src]allowlist_externals}
+
 [testenv:coverage]
```

Comment:
I get
```
$ ./sage -tox -e coverage.py-html
ROOT: HandledError| provided environments not found in configuration file:
coverage.py-html
```

Reply:
What does ` ./sage -tox --version` say on your system?

Reply:
https://github.com/tox-dev/tox/issues/3096 may be relevant for this.
I'll work around it.